synctl/cli.py

```diff
@@ -4683,6 +4779,8 @@ def main():
                 print('testid is required')
         elif get_args.op_type == POP_SIZE:
             pop_estimate.pop_size_estimate()
+        elif get_args.op_type == POP_COST:
```

Comment:
Use abbreviations here, also you can define a constant for cost.
```suggestion
        elif get_args.op_type == POP_COST or get_args.op_type == 'cost':
```